include/util/vector_view.hpp

```diff
@@ -166,6 +191,13 @@ template <> class vector_view<bool>
 
     bool operator[](const unsigned index) const { return at(index); }
 
+    reference operator[](const unsigned index)
+    {
+        const std::size_t bucket = index / UNSIGNED_BITS;
```

Comment:
Opposed to `at`, this one is not asserting the index anymore. Could you add an assert like in `at`?